src/test/java/com/ontotext/kafka/mocks/DummyProducer.java

```diff
@@ -6,10 +6,7 @@
 import org.apache.kafka.clients.producer.Producer;
 import org.apache.kafka.clients.producer.ProducerRecord;
 import org.apache.kafka.clients.producer.RecordMetadata;
-import org.apache.kafka.common.Metric;
-import org.apache.kafka.common.MetricName;
-import org.apache.kafka.common.PartitionInfo;
-import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.common.*;
```

Comment:
Wildcard import was blocked on some other Ontotext projects. Are we fine with it here, or it's just the IDE restructuring?

Reply:
I am using the IDE settings that the PRO team have, so I guess we are fine with it. Will double check just in case